### PR TITLE
[CORE-API] Create Place entity, route, controller and service #9

### PR DESCRIPTION
**Pull request by eduardosm7**
Solves #9

### CODE REVIEW

File: core-api/src/entities/place.entity.ts [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    private id: number;
+
+    @Column({ length: 64 })
+    @Length(3, 64, {
+        message: Messages.validation.place_name_length,
+    })
+    private name: string;
+
+    @Column()
+    private category_id: number;
```
Comment (gilesv) [inline core-api/src/entities/place.entity.ts]: This property should be camel-cased. `categoryId`

Comment (viniciusgiles) [inline core-api/src/entities/place.entity.ts]: This should be a relation, right?

Comment (viniciusgiles) [reply core-api/src/entities/place.entity.ts]: What if a place needs to have more than a single category?

Comment (eduardosm7) [reply core-api/src/entities/place.entity.ts]: Done

**Review by viniciusgiles — APPROVED** [review]

**Pull request merged**